content/pandas/concepts/dataframe/terms/isna/isna.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: '.isna()'
+Description: 'Checks whether the objects of a Dataframe or a series contain missing or null values and returns a corresponding boolean `True` or `False` value.'
+Subjects:
+    - 'Computer Science'
+    - 'Pandas'
+Tags:
+    - 'Methods'
+    - 'Data Science'
+CatalogContent:
+    - 'learn-python-3'
+    - 'paths/data-science'
+---
+
+
+The **`.isna()`** method checks whether the objects of a `Dataframe` or a series contain missing or null values `(NA, NaN)` and returns a new object with the same shape as the original object but with boolean values `True` or `False` as the elements. `True` indicates the presence of null or missing values and `False` indicates otherwise. The original `DataFrame` object, used to call the method, remains unchanged.
```

Comment:
```suggestion
The **`.isna()`** method checks whether the objects of a DataFrame or a Series contain missing or null values `(NA, NaN)` and returns a new object with the same shape as the original but with boolean values `True` or `False` as the elements. `True` indicates the presence of null or missing values and `False` indicates otherwise. The original DataFrame object, used to call the method, remains unchanged.
```